install_config/storage_examples/storage_classes_dynamic_provisioning.adoc

```diff
@@ -15,8 +15,13 @@ toc::[]
 [[install-config-storage-examples-storage-classes-dynamic-provisioning-Overview]]
 == Overview
 
-In these examples we will walk through a few scenarios of various configuratons of xref:../../install_config/persistent_storage/storage_classes.adoc#install-config-persistent-storage-storage-classes[StorageClasses] and Dynamic Provisioning using Google Cloud Platform Compute Engine (GCE).
-These examples assume some familiarity with Kubernetes, GCE and Persistent Disks and {product-title} is installed and xref:../../install_config/persistent_storage/persistent_storage_gce.adoc#install-config-persistent-storage-persistent-storage-gce[properly configured to use GCE].
+In these examples we will walk through a few scenarios of various configuratons of xref:../../install_config/persistent_storage/storage_classes.adoc#install-config-persistent-storage-storage-classes[StorageClasses] and Dynamic Provisioning using Google Cloud Platform Compute Engine (GCE). These examples
```

Comment:
"We" is first person. Should probably be something like:

"The examples in this topic show scenarios of various configurations..."